### PR TITLE
📄 Antitrust policy [due 23 Aug]

### PR DESCRIPTION
**Pull request by ebullient**
[🗳️ Vote progress](https://github.com/commonhaus/foundation/pull/188#issuecomment-2274635684 "IC_kwDOKRPTI86HlC-k")

Antitrust policy

akin to: https://www.linuxfoundation.org/legal/antitrust-policy

voting group: @commonhaus/cf-egc 

Do one of the following:

- Approve the PR or react with 👍 (:+1:) if it looks good to you
- Review with Comments or react with 👀 (:eyes:) if you're "ok" with it (it may not be your favorite)
- If you think it needs discussion or revision
    - Create a review, add your comments and require changes
    - Use the +- button to make a suggestion (instead of just adding a comment).

### CODE REVIEW

File: policies/antitrust-policy.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Participation in Foundation activities
+
+1. **Open participation**: The foundation encourages open participation in its projects and activities. Membership in CF should not be used to exclude or disadvantage any individual or entity.
+2. **Intellectual property**: Members should adhere to the [Intellectual Property Policy](./ip-policy.md), ensuring that participation in CF projects does not restrict competition or the free use of technologies.
+3. **Governance practices**: Project representatives must ensure that the governance practices used by their projects adhere to this anti-trust policy.
+
+## Handling Violations
+
+Any reported or suspected violations of this policy will be promptly investigated by the CF Council.
```
Comment (kenfinnigan) [inline policies/antitrust-policy.md:38]: Is there a particular place suspected violations should be reported?

Comment (ebullient) [reply policies/antitrust-policy.md:38]: `report@commonhaus.org`, as with all other things, I suspect. Want to lift that language from the conflict of interest policy and stick it in here?

Comment (kenfinnigan) [reply policies/antitrust-policy.md:38]:
```suggestion
Any reported or suspected violations of this policy will be promptly investigated by the CF Council.

### Reporting a Violation

Any suspected violations of this policy should be directed to the [report mailing list](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml)
```

**Review by henri-tremblay — APPROVED** [review]

**Review by cealsair — APPROVED** [review]

**Review by evanchooly — APPROVED** [review]

File: policies/antitrust-policy.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The Commonhaus Foundation (CF) requires that all members, activities, projects, committees, and working groups comply with applicable anti-trust laws.
+
+## Scope
+
+This policy applies to all participants in Commonhaus Foundation activities, including but not limited to general members, project representatives, councilors, and contributors to CF projects. It is designed to complement the [Conflict of Interest Policy](./conflict-of-interest.md) and other governance documents of the foundation.
+
+## Commitment to Compliance
+
+By participating in official or informal CF activities, participants commit to upholding this policy and promoting fair competition.
```
Comment (jkschneider) [inline policies/antitrust-policy.md:13]: "CF activities" is not clearly defined. Does this policy extend to the conduct of CF members and participants in every aspect of their lives?

Comment (ebullient) [reply policies/antitrust-policy.md:13]: Would something like this work better?

```suggestion
By participating in any official or informal activities of the Commonhaus Foundation that are directly related to its governance, projects, or operations, participants commit to upholding this policy and promoting fair competition.
```

**Comment by ebullient** [comment]
I have enough of a reset that I'm going to close this and re-open to reset the vote w/ a new timeline.

vote::result withdrawn

**Pull request closed without merging**